<!DOCTYPE html>
<html lang="en">
<head>
        <title>Resume</title>
        <meta charset="utf-8" />
        <link rel="stylesheet" href="http://schryer.github.io/schryer_pelican_blog/theme/css/main.css" type="text/css" />
   

        <!--[if IE]>
                <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->

        <!--[if lte IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="http://schryer.github.io/schryer_pelican_blog/css/ie.css"/>
                <script src="http://schryer.github.io/schryer_pelican_blog/js/IE8.js" type="text/javascript"></script><![endif]-->

        <!--[if lt IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="http://schryer.github.io/schryer_pelican_blog/css/ie6.css"/><![endif]-->

</head>

<body id="index" class="home">
        <header id="banner" class="body">
                <h1><a href="http://schryer.github.io/schryer_pelican_blog">David Schryer </a></h1>
                <nav><ul>
                    <li><a href="http://schryer.github.io/schryer_pelican_blog/pages/resume.html">Resume</a></li>
                    <li ><a href="http://schryer.github.io/schryer_pelican_blog/category/blog.html">Blog</a></li>
                    <li ><a href="http://schryer.github.io/schryer_pelican_blog/category/notebooks.html">Notebooks</a></li>
                </ul></nav>
        </header><!-- /#banner -->
        
<section id="content" class="body">    
    <h1 class="entry-title">Resume</h1>
    
<div class="document">


<div class="section" id="summary">
<h1>summary</h1>
<p class="myname">David Schryer</p>
<p class="tagline">Programmer — Data analyst — Technical editor/writer</p>
<p class="summary"><strong>Systems biologist/programmer with a chemical engineering background.</strong></p>
<p>I seek a position in a visionary company within a great team. Most
people would characterize me as an excellent communicator (founder of
<a class="reference external" href="http://scientificscribe.com">ScientificScribe</a>) with an eye for detail who enjoys tackling
difficult technical problems. I enjoy creating software and would like
an opportunity to work within a professional team.</p>
<!-- .. class:: shortcontact -->
<p><a class="reference external" href="mailto:schryer@gmail.com">schryer&#64;gmail.com</a> — (+372) 517 6922</p>
</div>
<div class="section" id="employment">
<h1>employment</h1>
<dl class="docutils">
<dt>University of Tartu,  Researcher <br /> 2012 — 2014.</dt>
<dd><ul class="first last simple">
<li>Analysis of label-free, dynamic <a class="reference external" href="http://en.wikipedia.org/wiki/Stable_isotope_labeling_by_amino_acids_in_cell_culture">SILAC</a>, and static <a class="reference external" href="http://en.wikipedia.org/wiki/Stable_isotope_labeling_by_amino_acids_in_cell_culture">SILAC</a> MS/MS proteomics data.</li>
<li>Custom analysis of sequencing data, including analysis of ribosome profiling data.</li>
<li>Instruction of PhD students in <a class="reference external" href="http://www.python.org">Python</a> data analysis and visualization techniques.</li>
</ul>
</dd>
<dt>Scientific Scribe OÜ,  Founder <br /> 2012.</dt>
<dd><ul class="first last simple">
<li>Technical editing of grant proposals, academic manuscripts, and PhD dissertations.</li>
<li>Typesetting and layout of complex documents in <a class="reference external" href="http://www.latex-project.org/">LaTeX</a>.</li>
</ul>
</dd>
<dt>Tallinn University of Technology, Institute of Cybernetics,  Researcher <br />  2008 – 2012.</dt>
<dd><ul class="first last simple">
<li>Development of tools to build and simulate isotopomer and isotopologue models.</li>
<li>Application of isotopomer modeling to study the recycling dynamics of phosphorylated metabolites in heart.</li>
<li>Flux Balance Analysis (<a class="reference external" href="http://en.wikipedia.org/wiki/Flux_Balance_Analysis">FBA</a>) of genome scale metabolic models.</li>
</ul>
</dd>
<dt>TFTAK, Systems biology developer <br /> 2006 – 2008.</dt>
<dd><ul class="first last simple">
<li>Developed computational tools for systems biology analysis.</li>
<li>Installed and improved the fermentation systems and laboratory infrastructure.</li>
<li>Introduced <a class="reference external" href="http://www.python.org">Python</a> to an MS Excel based analysis team.</li>
</ul>
</dd>
<dt>Tallinn University of Technology, Department of Chemical Engineering <br /> 2004 – 2006.</dt>
<dd><ul class="first last simple">
<li>Ran a pilot plant for the industrial extraction of protein from rapeseed.</li>
<li>Wrote an English language curriculum document for department accreditation.</li>
</ul>
</dd>
<dt>EBS high school, Science teacher <br /> 2005 – 2006.</dt>
<dd><ul class="first last simple">
<li>Developed and gave courses in Chemistry, Physics, and Computer Science.</li>
<li>Wrote an English language curriculum document for internal use at EBS.</li>
</ul>
</dd>
</dl>
<dl class="pagebreak docutils">
<dt>University of Toronto, Research assistant <br /> 2000 – 2004.</dt>
<dd><ul class="first last simple">
<li>Upgrade and repair of an X-ray photoelectron spectrometer (<a class="reference external" href="http://en.wikipedia.org/wiki/X-ray_photoelectron_spectroscopy">XPS</a>).</li>
<li>Maintenance of a custom mass spectrometer and associated <a class="reference external" href="http://en.wikipedia.org/wiki/Ultra-high_vacuum">UHV</a> systems.</li>
<li>Designed and built a multi-annulus flow apparatus used to measure the transport
properties of solid oxide fuel cell membranes at 1000°C.</li>
<li>Studied the kinetics of water exchange with cellulose using isotope tracers.</li>
<li>Developed a multimedia fate &amp; transport model to study persistent organic pollutants.</li>
</ul>
</dd>
</dl>
</div>
<div class="section" id="education">
<h1>education</h1>
<dl class="docutils">
<dt>Tallinn University of Technology, Doctor of philosophy <br /> Ph.D. in Technical Physics</dt>
<dd><ul class="first last simple">
<li>Graduation April 2012.</li>
<li>Dissertation — <em>Metabolic Flux Analysis of Compartmentalized Systems using Dynamic Isotopologue Modeling</em></li>
<li>Awards: (A) Archimedes Foundation Grant for PhD studies – $5904 per annum (B) DoRa travel awards for two Biophysical Society meetings – Below cost</li>
</ul>
</dd>
<dt>University of Toronto, Master of Applied Science <br /> M.A.Sc. in Chemical Engineering</dt>
<dd><ul class="first last simple">
<li>Graduation June 2005.</li>
<li>Dissertation — <em>On the determination of sub-second water–cellulose sorption kinetics using mass spectrometry</em></li>
<li>Awards: A) Ontario Graduate Scholarship in Science and Technology (OGSST) – $15000 per annum (B) University of Toronto Fellowship – $1200 (received twice)</li>
</ul>
</dd>
<dt>University of Toronto, Bachelor of Applied Science <br /> B.A.Sc. in Chemical Engineering</dt>
<dd><ul class="first last simple">
<li>Graduation June 2000.</li>
<li>Dissertation — <em>Measuring the ion mobility of oxygen through solid oxide membranes using transient</em> <sup>18</sup>O <em>exchange and</em> <a class="reference external" href="http://en.wikipedia.org/wiki/Secondary_ion_mass_spectrometry">SIMS</a> <em>depth profiling</em></li>
<li>Awards: (A) Best Plant Design Project 2000 (B) Undergraduate Achievement Scholarship – $1500 (received twice)</li>
</ul>
</dd>
</dl>
</div>
<div class="section" id="skills">
<h1>skills</h1>
<dl class="docutils">
<dt>Experienced programmer in:</dt>
<dd><ul class="first last simple">
<li><a class="reference external" href="http://www.python.org">Python</a> (package development, unit testing, reusable code)</li>
</ul>
</dd>
<dt>Visualization and analysis of data using:</dt>
<dd><ul class="first last simple">
<li><a class="reference external" href="http://www.python.org">Python</a> (matplotlib, pandas, NumPy, SciPy, BioPython, statsmodels)</li>
</ul>
</dd>
<dt>Development of complex documents, course material, and presentations:</dt>
<dd><ul class="first last simple">
<li><a class="reference external" href="http://www.latex-project.org/">LaTeX</a>, <a class="reference external" href="http://en.wikipedia.org/wiki/ReStructuredText">reST</a>, Sphinx, Django, Beamer, IPython notebook.</li>
</ul>
</dd>
<dt>Practices safe computation with version control and unit testing:</dt>
<dd><ul class="first last simple">
<li>hg, git, svn, bzr, nosetests</li>
</ul>
</dd>
</dl>
<dl class="pagebreak docutils">
<dt>Avid supporter of <a class="reference external" href="http://en.wikipedia.org/wiki/Free_and_open-source_software">FOSS</a> with an emphasis on sharing my code:</dt>
<dd><ul class="first last simple">
<li>Bitbucket (schryer), GitHub (schryer).</li>
</ul>
</dd>
<dt>Basic proficiency in:</dt>
<dd><ul class="first last simple">
<li>C, C++, Fortran, COBOL, Pascal, Java.</li>
</ul>
</dd>
<dt>Basic front-end and advanced back-end web development using:</dt>
<dd><ul class="first last simple">
<li>GAE, OpenShift, dotCloud, <a class="reference external" href="http://www.python.org">Python</a></li>
</ul>
</dd>
</dl>
</div>
<div class="section" id="publications">
<h1>publications</h1>
<ul class="simple">
<li>Kannan K, Kanabar P, Schryer DW, Florin T, Oh E, Bahroos N, Tenson
T, Weissman JS, Mankin AS: <em>The general mode of translation
inhibition by macrolide antibiotics.</em>
Proc Natl Acad Sci U S A. 2014 Nov 11;111(45):15958-63</li>
<li>Schryer DW, Peterson P, Illaste A, Vendelin M: <em>Sensitivity analysis of flux determination in heart by</em>
H<sub>2</sub><sup>18</sup>O <em>-provided labeling using a dynamic isotopologue model of energy transfer pathways.</em>
PLoS Computational Biology 2012, 8(12):e1002795</li>
<li>Schryer DW, Peterson P, Illaste A, Vendelin M: <em>Mathematical model of oxygen labeling to study heart energy transfer.</em>
Biophysical Journal, 2012, 102(3):141a</li>
<li>Schryer DW, Vendelin M, Peterson P: <em>Symbolic flux analysis for genome-scale metabolic networks.</em>
BMC Systems Biology 2011, 5:81.</li>
<li>Illaste A, Kalda M, Schryer DW, Sepp M: <em>Life of mice - development of cardiac energetics.</em>
The Journal of Physiology 2010, 588(Pt 23):4617–4619.</li>
<li>Illaste A, Schryer DW, Birkedal R, Peterson P, Vendelin M: <em>Determination of regional diffusion coefficients of fluorescent ATP in rat cardiomyocytes.</em>
Biophysical Journal 2010, 98(3):749a.</li>
<li>Schryer DW, Peterson P, Paalme T, Vendelin M: <em>Bidirectionality and compartmentation of metabolic fluxes are revealed in the dynamics of isotopomer networks.</em>
International Journal of Molecular Sciences 2009, 10(4):1697–1718.</li>
<li>Schryer DW, Peterson P, Paalme T, Vendelin M: <em>Isotopomeric</em> <sup>13</sup>C <em>labeling of amino acids reveal compartmentation in Saccharomyces uvarum.</em>
Biophysical Journal 2009, 96(3):308a.</li>
<li>Schryer DW, Bhavsar S, Diamond ML: <em>A preliminary model of the fate of geosmin &amp; MIB2 in Lake Ontario.</em>
In The Canadian Association on Water Quality Annual Meeting 2001.</li>
<li>Schryer DW, Diamond ML, Liu Q, Stern G, McCarry B: <em>Multimedia Urban Model: Validation and uncertainty analysis using a dynamic Monte Carlo method.</em>
In 22nd Annual SETAC meeting 2001.</li>
</ul>
</div>
<div class="section" id="contact">
<h1>contact</h1>
<ul class="contactlist simple">
<li><strong>email:</strong>  <a class="reference external" href="mailto:schryer@gmail.com">schryer&#64;gmail.com</a>, <a class="reference external" href="mailto:editor@scientificscribe.com">editor&#64;scientificscribe.com</a>, <a class="reference external" href="mailto:schryer@ut.ee">schryer&#64;ut.ee</a></li>
<li><strong>cell:</strong> (+372) 517 6922</li>
<li><strong>skype:</strong>  schryer</li>
<li><strong>linkedin:</strong>  <a class="reference external" href="http://www.linkedin.com/in/schryer">http://www.linkedin.com/in/schryer</a></li>
<li><strong>github:</strong> <a class="reference external" href="https://github.com/schryer">https://github.com/schryer</a></li>
<li><strong>bitbucket:</strong> <a class="reference external" href="https://bitbucket.org/schryer">https://bitbucket.org/schryer</a></li>
<li><strong>address:</strong> Sõle 19-12, Tallinn, Harjumaa, Estonia, 10614</li>
</ul>
<!-- - - - - -
links
- - - - -

  .. image:: python.png
     :width: 160px
     :height: 80px
     :target: http://www.python.com
     :align: left

  .. image:: logo.png
     :width: 150px
     :height: 80px
     :target: http://www.scientificscribe.com
     :align: right

  .. image:: github-logo.png
     :width: 90px
     :height: 80px
     :target: https://github.com/schryer
     :align: right

  .. image:: bitbucket-logo.png
     :width: 90px
     :height: 80px
     :target: https://bitbucket.org/schryer
     :align: right -->
</div>
</div>
<script type= "text/javascript">
    if (!document.getElementById('mathjaxscript_pelican_#%@#$@#')) {
        var mathjaxscript = document.createElement('script');
        mathjaxscript.id = 'mathjaxscript_pelican_#%@#$@#';
        mathjaxscript.type = 'text/javascript';
        mathjaxscript.src = 'https:' == document.location.protocol
                ? 'https://c328740.ssl.cf1.rackcdn.com/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
                : 'http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
        mathjaxscript[(window.opera ? "innerHTML" : "text")] =
            "MathJax.Hub.Config({" +
            "    config: ['MMLorHTML.js']," +
            "    TeX: { extensions: ['AMSmath.js','AMSsymbols.js','noErrors.js','noUndefined.js'], equationNumbers: { autoNumber: 'AMS' } }," +
            "    jax: ['input/TeX','input/MathML','output/HTML-CSS']," +
            "    extensions: ['tex2jax.js','mml2jax.js','MathMenu.js','MathZoom.js']," +
            "    displayAlign: 'center'," +
            "    displayIndent: '0em'," +
            "    showMathMenu: true," +
            "    tex2jax: { " +
            "        inlineMath: [ ['$','$'] ], " +
            "        displayMath: [ ['$$','$$'] ]," +
            "        processEscapes: true," +
            "        preview: 'TeX'," +
            "    }, " +
            "    'HTML-CSS': { " +
            "        styles: { '.MathJax_Display, .MathJax .mo, .MathJax .mi, .MathJax .mn': {color: 'black ! important'} }" +
            "    } " +
            "}); ";
        (document.body || document.getElementsByTagName('head')[0]).appendChild(mathjaxscript);
    }
</script>

</section>
        <section id="extras" class="body">
        </section><!-- /#extras -->

        <footer id="contentinfo" class="body">

        </footer><!-- /#contentinfo -->

</body>
</html>