<!DOCTYPE html>
<html lang="en">
<head>
        <title>Resume</title>
        <meta charset="utf-8" />
        <link rel="stylesheet" href="http://schryer.github.io/python_course_material/theme/css/main.css" type="text/css" />
   

        <!--[if IE]>
                <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->

        <!--[if lte IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="http://schryer.github.io/python_course_material/css/ie.css"/>
                <script src="http://schryer.github.io/python_course_material/js/IE8.js" type="text/javascript"></script><![endif]-->

        <!--[if lt IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="http://schryer.github.io/python_course_material/css/ie6.css"/><![endif]-->

</head>

<body id="index" class="home">
        <header id="banner" class="body">
                <h1><a href="http://schryer.github.io/python_course_material">David Schryer </a></h1>
                <nav><ul>
                    <li><a href="http://schryer.github.io/python_course_material/pages/resume.html">Resume</a></li>
                    <li ><a href="http://schryer.github.io/python_course_material/category/blog.html">Blog</a></li>
                    <li ><a href="http://schryer.github.io/python_course_material/category/notebooks.html">Notebooks</a></li>
                </ul></nav>
        </header><!-- /#banner -->
        
<section id="content" class="body">    
    <h1 class="entry-title">Resume</h1>
    <p>Researcher — Technical editor — Programmer — <a href="http://en.wikipedia.org/wiki/Free_and_open-source_software" title="Free and open source software">FOSS</a> activist</p>
<p><strong>Systems biology modeler with a chemical engineering background.</strong>
Specialty in dynamic isotope exchange experiments to study both
metabolic flux and protein production and turnover. Capable
communicator (founder of <a href="http://www.scientificscribe.com" title="Scientific Scribe OÜ">Scientific Scribe OÜ</a>) with modeling and
programming skills. Believes that reproducible research is best
realized using free and open source (<a href="http://en.wikipedia.org/wiki/Free_and_open-source_software" title="Free and open source software">FOSS</a>) tools.</p>
<p><em>Open to collaboration</em> — <a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#115;&#99;&#104;&#114;&#121;&#101;&#114;&#64;&#117;&#116;&#46;&#101;&#101;">&#115;&#99;&#104;&#114;&#121;&#101;&#114;&#64;&#117;&#116;&#46;&#101;&#101;</a></p>
<h2>Employment</h2>
<dl>
<dt>University of Tartu, Researcher, 2012.</dt>
<dd>
<ul>
<li>Analysis of label-free, dynamic <a href="http://en.wikipedia.org/wiki/Stable_isotope_labeling_by_amino_acids_in_cell_culture" title="Stable isotope labeling by amino acids in cell culture">SILAC</a>, and static <a href="http://en.wikipedia.org/wiki/Stable_isotope_labeling_by_amino_acids_in_cell_culture" title="Stable isotope labeling by amino acids in cell culture">SILAC</a>
    MS/MS proteomics data.</li>
<li>Processing sequencing data, including analysis of ribosome profiling data.</li>
<li>Instructing PhD students in data analysis and visualization (<a href="http://www.python.org" title="Python">Python</a>).</li>
</ul>
</dd>
<dt>Scientific Scribe OÜ, Founder, 2012.</dt>
<dd>
<ul>
<li>Technical editing of grant proposals and academic manuscripts.</li>
<li>Technical editing and typesetting of PhD dissertations in <a href="http://www.latex-project.org/" title="LaTeX">LaTeX</a>.</li>
</ul>
</dd>
<dt>Tallinn University of Technology, Institute of Cybernetics, Researcher, 2008 – 2012.</dt>
<dd>
<ul>
<li>Development of tools to build and simulate isotopomer and
    isotopologue models.</li>
<li>Application of isotopomer modeling to study the recycling
    dynamics of phosphorylated metabolites in heart.</li>
<li>Flux Balance Analysis (<a href="http://en.wikipedia.org/wiki/Flux_Balance_Analysis" title="Flux balance analysis">FBA</a>) of genome scale metabolic models.</li>
</ul>
</dd>
<dt>TFTAK, Systems biology developer, 2006 – 2008.</dt>
<dd>
<ul>
<li>Developed computational tools for systems biology analysis.</li>
<li>Installed and improved the fermentation systems and laboratory infrastructure.</li>
<li>Introduced <a href="http://www.python.org" title="Python">Python</a> to an MS Excel based analysis team.</li>
</ul>
</dd>
<dt>Tallinn University of Technology, Department of Chemical Engineering, 2004 – 2006.</dt>
<dd>
<ul>
<li>Ran a pilot plant for the industrial extraction of protein from
    rapeseed.</li>
<li>Wrote an English language curriculum document for department accreditation.</li>
</ul>
</dd>
<dt>EBS high school, Science teacher, 2005 – 2006.</dt>
<dd>
<ul>
<li>Developed and gave courses in Chemistry, Physics, and Computer
    Science.</li>
<li>Wrote an English language curriculum document for internal use at EBS.</li>
</ul>
</dd>
<dt>University of Toronto, Research assistant, 2000 – 2004.</dt>
<dd>
<ul>
<li>Upgrade and repair of an X-ray photoelectron spectrometer (<a href="http://en.wikipedia.org/wiki/X-ray_photoelectron_spectroscopy" title="X-ray photoelectron spectroscopy">XPS</a>).</li>
<li>Maintenance of a custom mass spectrometer and associated <a href="http://en.wikipedia.org/wiki/Ultra-high_vacuum" title="Ultra high vacuum">UHV</a> systems.</li>
<li>Designed and built a multi-annulus flow apparatus used to
    measure the transport properties of solid oxide fuel cell
    membranes at 1000°C.</li>
<li>Studied the kinetics of water exchange with cellulose using isotope tracers.</li>
<li>Developed a multimedia fate &amp; transport model to study persistent organic pollutants.</li>
</ul>
</dd>
</dl>
<h2>Education</h2>
<dl>
<dt>Tallinn University of Technology, Doctor of philosophy Ph.D. in Technical Physics</dt>
<dd>
<ul>
<li>Graduation April 2012.</li>
<li>Dissertation — <em>Metabolic Flux Analysis of Compartmentalized
    Systems using Dynamic Isotopologue Modeling</em></li>
<li>Awards: (A) Archimedes Foundation Grant for PhD studies – \$5904
    per annum (B) DoRa travel awards for two Biophysical Society
    meetings – Below cost</li>
</ul>
</dd>
<dt>University of Toronto, Master of Applied Science M.A.Sc. in Chemical Engineering</dt>
<dd>
<ul>
<li>Graduation June 2005.</li>
<li>Dissertation — <em>On the determination of sub-second
    water–cellulose sorption kinetics using mass spectrometry</em></li>
<li>Awards: A) Ontario Graduate Scholarship in Science and
    Technology (OGSST) – \$15000 per annum (B) University of Toronto
    Fellowship – \$1200 (received twice)</li>
</ul>
</dd>
<dt>University of Toronto, Bachelor of Applied Science B.A.Sc. in Chemical Engineering</dt>
<dd>
<ul>
<li>Graduation June 2000.</li>
<li>Dissertation — <em>Measuring the ion mobility of oxygen through
    solid oxide membranes using transient</em> $^{18}\text{O}$ <em>exchange and</em> <a href="http://en.wikipedia.org/wiki/Secondary_ion_mass_spectrometry" title="Secondary ion mass spectrometry">SIMS</a>
    <em>depth profiling</em></li>
<li>Awards: (A) Best Plant Design Project 2000 (B) Undergraduate
    Achievement Scholarship – \$1500 (received twice)</li>
</ul>
</dd>
</dl>
<h2>Skills</h2>
<dl>
<dt>Experienced programmer in:</dt>
<dd>
<ul>
<li><a href="http://www.python.org" title="Python">Python</a> (package development, unit testing, reusable code).</li>
</ul>
</dd>
<dt>Visualization and analysis of data using:</dt>
<dd>
<ul>
<li><a href="http://www.python.org" title="Python">Python</a> (matplotlib, pandas, NumPy, SciPy, BioPython, h5py).</li>
</ul>
</dd>
<dt>Development of complex documents, course material, and presentations:</dt>
<dd>
<ul>
<li><a href="http://www.latex-project.org/" title="LaTeX">LaTeX</a>, <a href="http://en.wikipedia.org/wiki/ReStructuredText" title="reStructuredText">reST</a>, Sphinx, Django, Beamer, <a href="http://ipython.org/notebook.html" title="IPython notebook">IPython notebook</a>.</li>
</ul>
</dd>
<dt>Practices safe computation with version control and unit testing:</dt>
<dd>
<ul>
<li>hg, git, svn, bzr, nosetests.</li>
</ul>
</dd>
<dt>Avid supporter of <a href="http://en.wikipedia.org/wiki/Free_and_open-source_software" title="Free and open source software">FOSS</a> with an emphasis on sharing my code:</dt>
<dd>
<ul>
<li>Bitbucket (<code>schryer</code>), GitHub (<code>schryer</code>).</li>
</ul>
</dd>
<dt>Basic proficiency in:</dt>
<dd>
<ul>
<li>C, C++, Fortran, COBOL, Pascal, Java.</li>
</ul>
</dd>
<dt>Basic front-end and advanced back-end web development using:</dt>
<dd>
<ul>
<li>GAE, OpenShift, dotCloud, <a href="http://www.python.org" title="Python">Python</a>.</li>
</ul>
</dd>
</dl>
<h2>Publications</h2>
<ul>
<li><strong>Schryer DW</strong>, Peterson P, Illaste A, Vendelin M: <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002795"><em>Sensitivity analysis
    of flux determination in heart by</em> $\text{H}_2^{18}\text{O}$ <em>-provided labeling
    using a dynamic isotopologue model of energy transfer pathways.</em></a></li>
<li><strong>Schryer DW</strong>, Peterson P, Illaste A, Vendelin M: <a href="http://www.sciencedirect.com/science/article/pii/S0006349511021254"><em>Mathematical model
    of oxygen labeling to study heart energy transfer.</em></a></li>
<li><strong>Schryer DW</strong>, Vendelin M, Peterson P: <a href="http://www.biomedcentral.com/1752-0509/5/81"><em>Symbolic flux analysis for
    genome-scale metabolic networks.</em></a></li>
<li>Illaste A, Kalda M, <strong>Schryer DW</strong>, Sepp M: <a href="http://jp.physoc.org/content/588/23/4617.long"><em>Life of mice - development
    of cardiac energetics.</em></a> </li>
<li>Illaste A, <strong>Schryer DW</strong>, Birkedal R, Peterson P, Vendelin M:
    <a href="http://www.sciencedirect.com/science/article/pii/S0006349509059128"><em>Determination of regional diffusion coefficients of fluorescent ATP
    in rat cardiomyocytes.</em></a> </li>
<li><strong>Schryer DW</strong>, Peterson P, Paalme T, Vendelin M: <a href="http://www.mdpi.com/1422-0067/10/4/1697"><em>Bidirectionality and
    compartmentation of metabolic fluxes are revealed in the dynamics of
    isotopomer networks.</em></a></li>
<li><strong>Schryer DW</strong>, Peterson P, Paalme T, Vendelin M: <a href="http://www.sciencedirect.com/science/article/pii/S0006349508017529"><em>Isotopomeric</em> $^{13}\text{C}$
    <em>labeling of amino acids reveal compartmentation in Saccharomyces
    uvarum.</em></a> </li>
<li><strong>Schryer DW</strong>, Bhavsar S, Diamond ML: <em>A preliminary model of the fate
    of geosmin &amp; MIB2 in Lake Ontario.</em> In The Canadian Association on
    Water Quality Annual Meeting 2001.</li>
<li><strong>Schryer DW</strong>, Diamond ML, Liu Q, Stern G, McCarry B: <em>Multimedia Urban
    Model: Validation and uncertainty analysis using a dynamic Monte
    Carlo method.</em> In 22nd Annual SETAC meeting 2001.</li>
</ul>
<h2>Contact</h2>
<ul>
<li><strong>github:</strong> <a href="https://schryer.github.io">https://schryer.github.io</a></li>
<li><strong>bitbucket:</strong> <a href="https://bitbucket.org/schryer">https://bitbucket.org/schryer</a></li>
</ul><script type= "text/javascript">
    if (!document.getElementById('mathjaxscript_pelican_#%@#$@#')) {
        var mathjaxscript = document.createElement('script');
        mathjaxscript.id = 'mathjaxscript_pelican_#%@#$@#';
        mathjaxscript.type = 'text/javascript';
        mathjaxscript.src = 'https:' == document.location.protocol
                ? 'https://c328740.ssl.cf1.rackcdn.com/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
                : 'http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
        mathjaxscript[(window.opera ? "innerHTML" : "text")] =
            "MathJax.Hub.Config({" +
            "    config: ['MMLorHTML.js']," +
            "    TeX: { extensions: ['AMSmath.js','AMSsymbols.js','noErrors.js','noUndefined.js'], equationNumbers: { autoNumber: 'AMS' } }," +
            "    jax: ['input/TeX','input/MathML','output/HTML-CSS']," +
            "    extensions: ['tex2jax.js','mml2jax.js','MathMenu.js','MathZoom.js']," +
            "    displayAlign: 'center'," +
            "    displayIndent: '0em'," +
            "    showMathMenu: true," +
            "    tex2jax: { " +
            "        inlineMath: [ ['$','$'] ], " +
            "        displayMath: [ ['$$','$$'] ]," +
            "        processEscapes: true," +
            "        preview: 'TeX'," +
            "    }, " +
            "    'HTML-CSS': { " +
            "        styles: { '.MathJax_Display, .MathJax .mo, .MathJax .mi, .MathJax .mn': {color: 'black ! important'} }" +
            "    } " +
            "}); ";
        (document.body || document.getElementsByTagName('head')[0]).appendChild(mathjaxscript);
    }
</script>

</section>
        <section id="extras" class="body">
        </section><!-- /#extras -->

        <footer id="contentinfo" class="body">

        </footer><!-- /#contentinfo -->

</body>
</html>